integrations/amazon_bedrock/pydoc/config.yml

```
@@ -0,0 +1,28 @@
+loaders:
+  - type: haystack_pydoc_tools.loaders.CustomPythonLoader
+    search_path: [../src]
+    modules: [
+      "haystack_integrations.components.generators.amazon_bedrock.generator",
```

I think we should document all files in the repo:

```suggestion
      "haystack_integrations.components.generators.amazon_bedrock.generator",
      "haystack_integrations.components.generators.amazon_bedrock.adapter",
      "haystack_integrations.components.generators.amazon_bedrock.errors",
      "haystack_integrations.components.generators.amazon_bedrock.handler",
```

---

@ZanSara added! (I added the import paths with the file names, e.g. adapter**s** and handler**s** (multiple), is that right 🤔 ?)

---

Oh that's right!